packages/build/src/test/type_test.ts

```diff
@@ -185,3 +183,44 @@ test("unsafeType", () => {
     anyOf: [{ type: "string" }, { type: "number" }],
   });
 });
+
+describe("array", () => {
+  test("no arguments", () => {
+    // @ts-expect-error no
+    assert.throws(() => array());
+  });
+
+  test("array of numbers", () => {
+    const arr1 = array("number");
+    // $ExpectType number[]
+    type t1 = ConvertBreadboardType<typeof arr1>;
```

Comment:
For these `defined but never used` errors you can add `/* eslint-disable @typescript-eslint/no-unused-vars */` blocks like in some of the above tests.

Reply:
ah I even saw yours, and then completely forgot